lib/assign-properties.js

```diff
@@ -1,7 +1,16 @@
-function giveMethodSuper(target, fn) {
+function giveMethodSuper(superclass, name, fn) {
+  var superFn = superclass[name];
+
+  if (typeof superFn !== 'function') {
+    superFn = function() {};
+  }
+
+  // jgwhite: Compatibility hack to allow `this._super.foo`
+  superFn[name] = superFn;
```

Comment:
its a bummer proxies don't really work everywhere, or we code use a proxy to add deprecation calls for when people use this....